.github/workflows/android.yml

```diff
@@ -0,0 +1,26 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: '**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: gradle
+
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: ./gradlew build
```

Comment:
You could consider if we needed build to the verification pull request.

Reply:
The build is verified every time a new Pull Request is created and once it is pushed to the default branch. It is better to be sure that branch in PR works before we can check it.

Reply:
We should make sure that unit tests are checked during the verification of PR, but if I'm not wrong `build` executes the unit tests

Reply:
Yes command **build** executes the _tests_ so if we want to use _assemble_ for pull requests also we need to check another job for _tests_

Reply:
The Build definitely includes the tests:
<img width="171" alt="image" src="https://user-images.githubusercontent.com/33199010/213859133-d9059c76-fd71-4c49-8287-d5a33152bb96.png">
@Nataniel-Antosik Is it possible to provide hard data on how beneficial creating assemble job instead of one build would be? Is it a common practice or just work around in case of significantly big projects?

Reply:
If we decide we don't want everything that offers *build* command we could split this commend on some jobs for the pull requests. I tested the command *build* and some jobs executed simultaneously doing the almost same work like *build* and splitting command *build* is faster. What about the common practice I believe it depends on the project.